salt/salt.spec

```diff
@@ -1603,18 +1611,6 @@ rm -f %{_localstatedir}/cache/salt/minion/thin/version
 %{_prefix}/lib/zypp/plugins/commit/zyppnotify
 %endif
 
-# Install Yum plugins only on RH machines
-%if 0%{?fedora} || 0%{?rhel}
-%if 0%{?fedora} >= 22 || 0%{?rhel} >= 8
-%{python3_sitelib}/dnf-plugins/dnfnotify.py
-%{python3_sitelib}/dnf-plugins/__pycache__/dnfnotify.*
-%{_sysconfdir}/dnf/plugins/dnfnotify.conf
-%else
-%{_prefix}/share/yum-plugins/yumnotify.*
-%{_sysconfdir}/yum/pluginconf.d/yumnotify.conf
-%endif
-%endif
-
```

Comment:
Just checked and it seems that dnfnotfy is using platform python, so no any sense to move it into `python3xx-salt` and seems we can drop `yumnotify` at all as RH8 has dnf by default and we are not building the sources for lower versions than 8

Reply:
Actually, we do not build the classic package for RH lower than version 8, but we might still need it for the Salt Bundle, so we should not drop this from the Salt codebase yet.

Reply:
For the placement of the dnfplugin in the exact subpackage the tricky point that actually python3-salt is intended to be used with platform python which is the same in this case which will run dnfnotify, but we are changing the placement of the file in comparison to the previous packages shipped, maybe not a big deal here, but it's not really required in the scope of this change. So I'm bit confused here finally.